ui/src/widgets/ui-slider/UISlider.vue

```diff
@@ -168,6 +168,15 @@ export default {
             if (typeof updates.iconPrepend !== 'undefined') {
                 this.dynamic.iconPrepend = updates.iconPrepend
             }
+            if (typeof updates.color !== 'undefined') {
```

Comment:
@arturv2000 if we're considering truthy values including "null" as well as "undefined" if we can simply use as below

`if (updates.color) {`

instead of

`if (typeof updates.color !== 'undefined') {`

Anyway it's based on the approach

Reply:
Thanks for the feedback,

Considering the use context of `ui_update` values i would say a better approach would be:

```js
if('color' in updates){
```

We may want to (let's say disable the property to attempt to return to default value). At least `null` is a valid value, is the way to return the variable/property to the default widget configuration value.

Personally, i don't like to use
```js
if (updates.color)
```

Since sometimes, that habit makes we use it in situations where a `false`, `null` or empty string may be valid values.

Reply:
Thanks for the clarification